nodes/src/nodes/chunker/chunker_strategies.py

```
@@ -0,0 +1,381 @@
+
+            result.append(
+                {
+                    'text': chunk_text,
+                    'metadata': {
+                        'chunk_index': chunk_index,
+                        'start_char': start_char,
+                        'end_char': end_char,
+                    },
+                }
+            )
+            chunk_index += 1
```

Ruff (0.15.10)

[warning] Use `enumerate()` for index variable `chunk_index` in `for` loop

(SIM113)